user_docs/en/changes.t2t

```diff
@@ -105,6 +105,7 @@ Users of Poedit 1 are encouraged to update to Poedit 3 if they want to rely on e
   - When reinstalling an incompatible add-on it is no longer forcefully disabled. (#15584, @lukaszgo1)
   - Disabled and incompatible add-ons can now be updated. (#15568, #15029)
   - NVDA now recovers and displays an error in a case where an add-on fails to download correctly. (#15796)
+  - When pressing ctrl+tab, focus properly moves to the new current tab title. (#14986, @ABuffEr)

```

Comment:
this PR is targeting master and should go into 2024.2. Please move the change log to there.

Reply:
My mistake, looking now at branches available I suppose to have to do a PR targeting beta; if you'll accept it I close this and open another PR. I'll move changelog line otherwise, but 2024.1 would be preferable.

Reply:
2024.1 is already deep into beta stage - it is too late for risky changes of behaviour - particularly this case where it is generally agreed that it might not be the final behaviour. I believe the behaviour included in this PR was part of the original add-on store PR (if not, it was something similar) which was reverted to the current behaviour based on feedback.

Reply:
I strongly doubt that there was a feedback against this feature, I'm quite sure it's an accessibility issue of that wx control from a long time ago (I mean, at least since we changed settings structure years and years ago).
Anyway, ok, I disagree but will change release target in changelog with next commit. Move it as needed if discussion proceeds.

Reply:
For reference, the prototype set focus to the add-on list view, this was later reverted due to feedback and the comment/issue was created to find a new solution.

- https://github.com/nvaccess/nvda/pull/13985/commits/7009a952762ad6b8e9a6169321bb77d822da6369
- https://github.com/nvaccess/nvda/pull/13985/commits/643fa00facfa9fa2223b6323ef9569634c05834f
- https://github.com/nvaccess/nvda/discussions/14912#discussioncomment-6035742